src/transformers/configuration_utils.py

```diff
@@ -195,6 +196,8 @@ def __init__(self, **kwargs):
         self.bos_token_id = kwargs.pop("bos_token_id", None)
         self.pad_token_id = kwargs.pop("pad_token_id", None)
         self.eos_token_id = kwargs.pop("eos_token_id", None)
+        self.sep_token_id = kwargs.pop("sep_token_id", None)
```

Comment:
I don't understand how this is related.

Reply:
It's not - I sneakily tried to squeeze it in this PR. I think the `sep_token_id` should go directly to `configuration_utils.py` similar to `eos_token_id`, `pad_token_id` and `bos_token_id`